.github/workflows/spelling.yml

```diff
@@ -15,9 +15,6 @@ on:
     - 'opened'
     - 'reopened'
     - 'synchronize'
-  issue_comment:
```

Comment:
we missed line 74, it should also be changed  😄 

```diff
diff --git a/.github/workflows/spelling.yml b/.github/workflows/spelling.yml
index 2696e09..d594e20 100644
--- a/.github/workflows/spelling.yml
+++ b/.github/workflows/spelling.yml
@@ -74,7 +74,7 @@ jobs:
     if: (success() || failure()) && needs.spelling.outputs.followup && contains(github.event_name, 'pull_request')
     steps:
     - name: comment
-      uses: check-spelling/check-spelling@v0.0.21-alpha1
+      uses: check-spelling/check-spelling@prerelease
       with:
         checkout: true
         task: ${{ needs.spelling.outputs.followup }}
```

Reply:
it should be `v0.0.21-alpha1`, the maintainer created it just for us

Reply:
done